### DISCUSSION TITLE
What is the best way to update Nodejs

### DISCUSSION THREAD

**Discussion by farvashani** [Tecnativa/doodba#416] [Q&A]
What is the best way to update nodejs in doodba docker image for odoo 14.0?

**Comment by joao-p-marques** [comment]
Hi @farvashani 

You can add `nodejs` to the `custom/dependencies/apt.txt`. As explained [here](https://github.com/Tecnativa/doodba#optodoocustomdependenciestxt), those files are used to indicate dependencies you wish to install in your subimage, and Doodba reads them and does that automatically. When it tries to install `nodejs`, it will automatically update.
